packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -7,12 +7,26 @@ const cli = {
     yargs.options({
       c: {
         type: 'string',
-        alias: 'configuration',
+        alias: 'config',
         describe: 'A file path to the test runner configuration.',
       },
+      formFactors: {
+        type: 'array',
+        describe: 'A list of form factors for the test run.',
```

Comment:
Do we want to set a default formFactor of `huge`?

Reply:
No, we're unfortunately in a bit of on odd situation with the form factor. Defaulting to `huge` here will set the `process.env.FORM_FACTOR` to `huge` which will filter all the `describeViewports` tests. Setting no default here will default to huge in the service's before hook, but allow other form factors in `describeViewports`